salt/proxy/junos.py

```diff
@@ -55,14 +58,17 @@
     )
     from ncclient.operations.errors import TimeoutExpiredError
     from ncclient.transport.third_party.junos.ioproc import IOProc
-except ImportError:
+
+## DGM except ImportError:
+except ImportError as ex:
+    log.error("DGM proxy junos failed to import, exception {}".format(str(ex)))
     HAS_JUNOS = False
 
 __proxyenabled__ = ["junos"]
 
 thisproxy = {}
 
-log = logging.getLogger(__name__)
+## DGM log = logging.getLogger(__name__)
 
```

Comment:
Maybe I didn't understand what DGM is after all...

Reply:
Dam, missed one, will fix, thanks for catching my sloppiness